### PR TITLE
Assign the owner as a label to the project

### PR DESCRIPTION
**Pull request by felipecruz91**
Refers to https://github.com/opendevstack/ods-provisioning-app/issues/668

### CODE REVIEW

File: create-projects/create-projects.sh [hunk 1]
```diff
@@ -80,6 +80,17 @@ if [ -n "${PROJECT_ADMINS}" ]; then
     oc policy add-role-to-user admin "${admin_user}" -n "${PROJECT_ID}-test"
     oc policy add-role-to-user admin "${admin_user}" -n "${PROJECT_ID}-cd"
   done
+  
+  echo "Assign the owner as a label to the OpenShift project"
+  # Take the first in the list as owner of the project.
+  # Labels do not allow the '@' char so let's replace it with '_at_'
+  # For instance, foo@bar.com will be converted to foo_at_bar.com
+  # resulting in the following label: ods.project.owner=foot_at_bar.com
+  namespace_owner=$(echo "${PROJECT_ADMINS}" | cut -d "," -f1 | sed -r 's/@/_at_/g')
```
Comment (felipecruz91) [inline create-projects/create-projects.sh:89]: I'm assuming `PROJECT_ADMINS` is a comma-separated list of email addresses, could anyone confirm?

Comment (michaelsauter) [reply create-projects/create-projects.sh:89]: There are tests for this :) https://github.com/opendevstack/ods-core/blob/master/create-projects/tests/run.sh

**Comment by metmajer** [comment]
@felipecruz91 would be great to have this in ODS 4.0. Any open questions?

**Comment by felipecruz91** [comment]
@metmajer No questions from my side. I just have to update the PR to reflect the new label name suggested by @segator here:
 https://github.com/opendevstack/ods-provisioning-app/issues/668#issuecomment-769044420

**Review by stitakis — COMMENTED** [review]
looks good to me

**Review by stitakis — DISMISSED** [review]

**Review by metmajer — DISMISSED** [review]
👍

**Comment by metmajer** [comment]
@felipecruz91 please resolve the conflicts, then squash and merge into a single commit. Thanks

**Pull request merged**